### ISSUE TITLE
Suggestion: Reduce Marble Cave Resistance to -3

### ISSUE THREAD

**Issue by Ianshogren** [xJon/Tekkit-2#50]
Problem: Marble caves do not generate frequent enough.

Solution: Project Red has an integer being added to the base marble cave resistance suggesting that 0 is not the lowest value but rather -4. Upon testing more marble caves generate when the resistance is lowered. 
Additionally, forking PR and modifying the cluster size and min / max Y produces larger and more frequent marble caves although this will require authorization from MrTJP through a pull request. He would be more likely to approve the request if modifying the clusterSize and Min/Max Y were made possible by the configuration file so that the default generation has not been changed.

Solution TLDR: Changing the resistance value to -3 while increasing the clusterSize and slightly increasing the max Y made finding marble caves more similar to Redpower 2.

From:
/MrTJP/ProjectRed/tree/1.12.x/src/mrtjp/projectred/exploration)/proxies.scala
```
   //Marble
        if (Configurator.gen_MarbleCave)
        {
            val logic = new GenLogicUniform
            logic.name = "pr_marblecave"
            logic.resistance = 4+Configurator.gen_MarbleCave_resistance
            logic.allowRetroGen = Configurator.gen_MarbleCave_retro
            logic.dimensionBlacklist = false
            logic.dimensions = Set(0)
            logic.minY = 32
            logic.maxY = 64
            val gen = new WorldGenCaveReformer
            gen.cluster = Set(((blockDecorativeStone, DecorativeStoneDefs.MARBLE.meta), 1))
            gen.clusterSize = 4096
            gen.material = Set((Blocks.STONE, 0))
            logic.gen = gen
            SimpleGenHandler.registerStructure(logic)
        }
```

**Comment by xJon** [comment]
This was actually an issue with the gem generation, which I ended up disabling in Project Red and using CoFH World instead. Didn't know negative values would be a workaround, will check it out!

**Comment by Ianshogren** [comment]
> This was actually an issue with the gem generation, which I ended up disabling in Project Red and using CoFH World instead. Didn't know negative values would be a workaround, will check it out! Can I ask if you ran some tests yourself using the negative value?

Yes, but really only a quick glance with spectator mode and night vision. I have an local fork of PR with a modified clusterSize, max Y, and removed the resistance offset. Only modifying the resistance to negative values (such as -3 and -4) produces more frequent caves but they remain small. If you are not able to rebuild the mod I suggest play testing around with -3 and -4 to find something that is not too common but still remains interesting.

**Comment by xJon** [comment]
@Ianshogren As the license is still not open I wouldn't consider forking and rebuilding but changing the configuration value to a negative one is an option as long as it doesn't present new issues.

**Comment by xJon** [comment]
Included in `1.0.2`, it seems to work well! Thanks again for the suggestion :)
